donut/templates/includes/header.html

```diff
@@ -27,6 +27,7 @@
                         <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">User Services <span class="caret"></span></a>
                         <ul class="dropdown-menu" role="menu">
                             <li><a href="#">ARC Feedback</a></li>
+			    <li><a href="{{ url_for('bodfeedback.bodfeedback') }}">BoD Feedback</a></li>
```

Comment:
previously the bod feedback form was linked under ASCIT -> Suggestions. If you think it makes more sense to put it here, you should remove that one. otherwise remove this and link that